pkg/interop/interop.go

```diff
@@ -92,9 +110,65 @@ func setupLogging(logger *log.Logger) {
       0666,
     )
     if err != nil {
-      log.Infof("failed to log to file, using default stderr: %s", err)
+      log.Warnf("failed to log to file, using default stderr: %s", err)
     } else {
       logger.Out = file
     }
   }
+
+  i.Logger = logger
+}
+
+func setupClient(i *Interop) error {
+  licenseKey := viper.GetString("licenseKey")
+  if licenseKey == "" {
+    licenseKey = os.Getenv("NEW_RELIC_LICENSE_KEY")
+    if licenseKey == "" {
+      return fmt.Errorf("missing New Relic license key")
+    }
+  }
```

Comment:
I don't really think we need a license key unless we are using events but can leave this for now.

Reply:
ok